Vagrantfile

```diff
@@ -20,25 +20,26 @@ Vagrant.configure(2) do |config|
   ##
   ##  puppet4master
   ##
-  # The "puppet4master" string is the name of the box. hence you can do "vagrant up puppet4444master"
+  # The "puppet4master" string is the name of the box hence you can do "vagrant up puppet4master"
   config.vm.define "puppet4master" do |puppet4master_config|
-    puppet4master_config.vm.box = "centos/7"
+
+    # Box used is https://github.com/holms/vagrant-centos7-box/releases/download/7.1.1503.001/CentOS-7.1.1503-x86_64-netboot.box
+    puppet4master_config.vm.box = "CentOS7.0"
```

Comment:
See the above comment - that is the https link. Could do vagrant box add https link to avoid pulling it down from the internet every time.